_shared/project/.gitignore

```diff
@@ -10,6 +10,7 @@ supervisord.log
 supervisord.pid
 .DS_Store
 .devdata.env
+.eslintcache
```

Comment:
`.eslintcache` needs to be `.gitignore`'d but was missing from the cookiecutter's `.gitignore` template